tash

```diff
@@ -0,0 +1,181 @@
+[33mcommit 3fc09cbe568adc54386fabb9e32a80e298ea9a67[m[33m ([m[1;36mHEAD -> [m[1;32mprojects_details_grid[m[33m, [m[1;31morigin/projects_details_grid[m[33m)[m
```

Comment:
nie wiem co to za plik z gita, ale nie powinien się tu znaleźć :)
dodaj go do gitignore

Reply:
git log - cos poklikałem i widocznie zapisało ;)